Classes/Controller/FeUsersController.php

```diff
@@ -117,7 +116,9 @@ public function renderAjax(ServerRequestInterface $request): Response
 			if (isset($row['usergroup'])) {
```

Comment:
`usergroup` is _always_ set when it is a fe user, suspect a check could be done here about it being empty